model/src/main/kotlin/config/AnalyzerConfiguration.kt

```diff
@@ -48,7 +48,12 @@ data class AnalyzerConfiguration(
      * Package manager specific configurations. The key needs to match the name of the package manager class, e.g.
      * "NuGet" for the NuGet package manager.
      */
-    val packageManagers: Map<String, PackageManagerConfiguration>? = null
+    val packageManagers: Map<String, PackageManagerConfiguration>? = null,
+
+    /**
+     * A flag to control whether excluded scopes and paths should be skipped during the analysis.
+     */
+    val skipExcluded: Boolean = false
```

Comment:
Please document this new option. I guess the best place to do this would be this file:
https://github.com/oss-review-toolkit/ort/blob/main/docs/config-file-ort-yml.md#excludes

In there, you'd also need to adjust this sentence, as this changes with this PR:
> Note that the excluded parts are analyzed and scanned

Reply:
Sure, this is on my agenda, but I want to get this merged first to be sure that there are no more changes in naming or otherwise. I will then open a follow-up PR.